packages/calcite-components/src/components/switch/switch.stories.ts

```diff
@@ -35,3 +35,14 @@ export const darkModeRTL_TestOnly = (): string => html`
 darkModeRTL_TestOnly.parameters = { modes: modesDarkDefault };
 
 export const disabled_TestOnly = (): string => html`<calcite-switch disabled checked></calcite-switch>`;
+
+export const Focus_TestOnly = (): string =>
```

Comment:
I think you may need to add a delay to the story to ensure the screenshot is taken when the test setup is done. 

@benelan do you have Any additional tips?

Reply:
It’s currently not showing the focus outline. 

![1FE0E813-3A0C-4D40-BFDC-2F690F88BF8D](https://github.com/Esri/calcite-design-system/assets/197440/47a85c83-b7d5-4727-a47d-e933c5049674)